src/shared/api/server/summary.ts

```diff
@@ -56,11 +57,43 @@ export async function GET(req: NextRequest): Promise<NextResponse<SummaryRespons
 
   const summary = results[0];
   if (!summary) {
-    return NextResponse.json(
-      { error: `No summary found for ${baseAssetSymbol}/${quoteAssetSymbol}` },
-      { status: 400 },
-    );
+    return NextResponse.json({ window: durationWindow, noData: true });
   }
 
-  return NextResponse.json(summary);
+  const priceDiff = summary.price - summary.price_then;
+  const change = {
+    value: calculateDisplayPrice(priceDiff, baseAssetMetadata, quoteAssetMetadata),
+    sign: priceDiffLabel(priceDiff),
+    percent:
+      summary.price === 0
+        ? '0'
+        : round({
+            value: Math.abs(((summary.price - summary.price_then) / summary.price_then) * 100),
+            decimals: 2,
+          }),
+  };
+
+  const dataResponse = new SummaryDataResponse({
+    window: durationWindow,
+    directVolume: toValueView({
+      amount: summary.direct_volume_over_window,
+      metadata: quoteAssetMetadata,
+    }),
```

Comment:
Passing volume as a `ValueView` allows us to use the normal `ValueViewComponent` that handles the display exponents correctly automatically.